src/generate-grpc-web.ts

```diff
@@ -173,7 +173,7 @@ function generateGrpcWebRpcType(ctx: Context, returnObservable: boolean, hasStre
   const { options } = ctx;
   const { useAbortSignal } = options;
 
-  chunks.push(code`interface Rpc {`);
+  chunks.push(code`export interface Rpc {`);
```

Comment:
Hi @marcaddeo , thanks for the PR!

Can you update this to conditionally add the `export`, i.e. something like:

```ts
  const maybeExport = options.exportCommonSymbols ? "export" : "";
```

Because we have some users where, when working with multiple `*.proto` files, these common exports can import conflicts, so they can opt to turn off the export. Thanks!

Reply:
Ah dang, that's true. I'm one of those people who opt out of the common exports too... So now this isn't going to help me at all 😅 

Will update though.

Reply:
Oh no! Sorry! :-)

Fwiw #1109 is an idea to make a shared `./ts-proto-types.ts` type file that have all these common types, and be output all the time, in just one spot, in theory allowing us to kill the common exports problem.

If you're up for tackling a PR for that, that'd be amazing! :pray: